### DISCUSSION TITLE
Using `pixels` without displaying a window - is there a "Headless mode"?

### DISCUSSION THREAD

**Discussion by lewisfm** [parasyte/pixels#408] [Q&A]
Is it possible to draw something using `pixels` without presenting a window to the user?

**Comment by parasyte** [comment]
The only thing that `pixels` gives you is the ability to display a texture on a window. If you don't want to do that, then pixel plotting to a `[u8; width * height * 4]` or `Vec<u8>` is "using `pixels` without displaying a window".

I would like to hear what the use case is and why you believe `pixels` would help.
